database/factories/LanguageLineFactory.php

```diff
@@ -14,8 +16,30 @@ class LanguageLineFactory extends Factory
      *
      * @return array<string, mixed>
      */
-    public function definition()
+    public function definition(): array
```

Comment:
I went along our preferred practice on other projects and suggested minimalist yet ready-to-use factories.